### PR TITLE
[DOCs/operators]: Create a Nym ISP sheet, write gateway-probe guide & clean FnF docs

### PR DESCRIPTION
**Pull request by serinko**
# Nym version of Tor's Good Bad ISPs

https://community.torproject.org/relay/community-resources/good-bad-isps/

**Create ISP list tailored for Nym:**

- [x] Initialise data/isp-sheet.csv
- [x] Add findings by LunarDAO
- [x] write a python script to render csv to mrkdown
- [x] Troubleshoot Python modules envs on CI/CD runners
- [x] add args fn for sorting the sheet
- [x] create a page in community councel with cmdrun of above
- [x] edit how to add info part
- [x] share with the operator community

**Create gateway-probe guide**

- [x] Dependencies
- [x] How to build the binary
- [x] Command Examples 

**Cleanup**

- [x] FnF cleanup
- [x] Merge performance and monitoring pages
- [x] Add probe to the list

### CODE REVIEW

File: documentation/operators/src/legal/isp-list.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Where to host your `nym-node`?
+
+```admonish info
```
Comment (mfahampshire) [inline documentation/operators/src/legal/isp-list.md:3]: why do we need this if we already have this license for the rest of the docs?

File: documentation/operators/src/legal/isp-list.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+The entire content of this page is under [Creative Commons Attribution 4.0 International Public License](https://creativecommons.org/licenses/by/4.0/).
+```
+
+Inspired by a valubale resource, done by Tor community - [*Good Bad ISPs*](https://community.torproject.org/relay/community-resources/good-bad-isps/), LunarDAO squad initiated a table customised for Nym Exit Gateways operators.
```
Comment (mfahampshire) [inline documentation/operators/src/legal/isp-list.md]: spelling mistake in "valuable"

File: documentation/operators/src/testing/gateway-probe.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# Nym Gateway Probe
+
+Nym Node operators running Gateway functionality are already familiar with the monitoring tool [Harbourmaster.nymtech.net](https://harbourmaster.nymtech.net). Under the hood of Nym Harbourmaster runs iterations of `nym-gateway-probe` doing various checks and displaying the results on the interface. Operators don't have operators rely on the probe run by Nym and wait for the data refresh. With `nym-gateway-probe` everyone can check any Gateway on the network. In one command the client queries data from:
```
Comment (mfahampshire) [inline documentation/operators/src/testing/gateway-probe.md]: " Operators don't have operators rely on the probe run by Nym and wait for the data refresh." 

What do you mean here?

**Review by mfahampshire — CHANGES_REQUESTED** [review]
minor stuff otherwise looks great

File: documentation/operators/src/legal/isp-list.md (hunk 1 above)
Comment (serinko) [reply documentation/operators/src/legal/isp-list.md:3]: true, there was something with the community legal pages etc to be more xplicit, but I am happy to take it down. your call.

File: documentation/operators/src/testing/gateway-probe.md (hunk 3 above)
Comment (serinko) [reply documentation/operators/src/testing/gateway-probe.md]: typo, fixing

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**